src/components/settings-page/index.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import PageContainer from '../page-container'
+
+import { StoreType, CategoryType } from '../../types'
+
+import './styles.scss'
+
+export default class Settings extends React.Component {
+  constructor(props) {
+    super(props)
+    this.steps = [
```

Comment:
Pourquoi ça et pas dans ton render un switch avec le call vers le bon render à chaque fois ?

Reply:
C'est toi qui l'a fais ça je crois

Reply:
Change le alors 😉

Reply:
Copy That

Reply:
Hmm je viens de regarder ça, je peux tout enlever et écrire en dur mais ça te permet de générer le menu avec les liens automatiquement sur le côté et tout donc je sais pas si tu veux le laisser ou que je le refasse ?

Reply:
Laisse le